library/general/test/test_helper.rb

```diff
@@ -1,4 +1,10 @@
-
 top_srcdir = File.expand_path("../../../..", __FILE__)
 inc_dirs = Dir.glob("#{top_srcdir}/library/*/src")
 ENV["Y2DIR"] = inc_dirs.join(":")
+
+require "yast"
+require_relative "SCRStub"
+
+RSpec.configure do |c|
+  c.include SCRStub
+end
```

Comment:
I think this part should be in `SCRStub` so when you require it, it is automatically assigned to RSpec. It will tight SCRStub to rspec, but it on other hand allow one place where we maintain connection and we do not need to fix all test_helper when rspec changes.